modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectDefinitionResourceTest.java

```diff
@@ -399,6 +407,38 @@ public void testPostObjectDefinition() throws Exception {
 			testPostObjectDefinition_addObjectDefinition(
 				randomObjectDefinition);
 
+		JSONObject jsonObject = HTTPTestUtil.invokeToJSONObject(
+			null,
+			StringBundler.concat(
+				"object-admin/v1.0/object-definitions",
+				"/by-external-reference-code/",
+				postObjectDefinition.getExternalReferenceCode(),
+				"?nestedFields=permissions"),
+			Http.Method.GET);
+
+		JSONArray jsonArray = jsonObject.getJSONArray("permissions");
+
+		jsonObject = jsonArray.getJSONObject(0);
+
+		Role ownerRole = _roleLocalService.getRole(
+			TestPropsValues.getCompanyId(), RoleConstants.OWNER);
+
+		JSONAssert.assertEquals(
+			JSONUtil.put(
+				"actionIds",
+				new String[] {
+					ActionKeys.DELETE, ActionKeys.PERMISSIONS,
+					ActionKeys.UPDATE, ActionKeys.VIEW
+				}
+			).put(
+				"roleExternalReferenceCode", ownerRole.getExternalReferenceCode()
+			).put(
+				"roleName", ownerRole.getName()
+			).put(
+				"roleType", ownerRole.getTypeLabel()
+			).toString(),
+			jsonObject.toString(), JSONCompareMode.LENIENT);
+
```

Comment:
@vngoliveira  could we define a new comment block separated to permissions? 

// Permissions

....